azavea/django-amazon-ses
# Migrate from Travis to GitHub Actions

## Overview

Ports Travis CI config to a GitHub Actions workflow.

Closes #67  

### Notes

The `PYPI_AZAVEA_USERNAME` and `PYPI_AZAVEA_PASSWORD` secrets have been made available to this repository.

## Testing Instructions

 * Verify that the [GitHub Actions CI workflow](https://github.com/azavea/django-amazon-ses/actions/runs/206073800) passes.
* Verify that [this Release workflow](https://github.com/azavea/django-amazon-ses/runs/977540454?check_suite_focus=true) ran. The workflow was expected to fail at the "Build and publish package" step since the PyPI credentials had not been shared with this repository yet.
